Vizzuality/i2i
# Feature/sidebar icons

Description of the PR

New styles for tabs in map sidebar (data-portal, geospatial data). Icons added and styles adapted

## Testing instructions

Go to data-portal click geospatial data, click on any of the flags, check sidebar tabs
eg http://localhost:3000/data-portal/BGD/fsp-maps?lat=0&lng=0&zoom=3

## Pivotal Tracker

https://www.pivotaltracker.com/story/show/168661172
